src/BSML/Tags/ScrollViewTag.cpp

```diff
@@ -29,7 +29,7 @@ namespace BSML {
     HMUI::TextPageScrollView* get_scrollViewTemplate() {
         static SafePtrUnity<HMUI::TextPageScrollView> scrollViewTemplate;
         if (!scrollViewTemplate) {
-            scrollViewTemplate = UnityEngine::Resources::FindObjectsOfTypeAll<GlobalNamespace::ReleaseInfoViewController*>()->FirstOrDefault()->_textPageScrollView;
+            scrollViewTemplate = UnityEngine::Object::FindObjectOfType<GlobalNamespace::EulaDisplayViewController*>(true)->_textPageScrollView;
```

Comment:
`resources findobjectsoftypeall` and `object findobjectoftype(true)` do the same thing